render.c

```diff
@@ -26,39 +27,49 @@ void render(struct slurp_output *output) {
 	cairo_paint(cairo);
 
 	struct slurp_seat *seat;
-	wl_list_for_each(seat, &state->seats, link) {
-		if (!seat->wl_pointer) continue;
-		if (!seat->has_selection) {
+	wl_list_for_each(seat, &state->seats, link)
+	{
+		struct slurp_selection *current_selection =
+			seat_get_current_selection(seat);
+		if (current_selection == NULL)
+			continue;
+		if (!seat->wl_pointer)
```

Comment:
If the seat has a touch device but no pointer device, we should still draw the selection. So I think we should drop this check.

Reply:
I think by this and previous comment I'll rewrite or even remove the function entirely.